packages/core/deploy/015_deploy_emp_creator.js

```diff
@@ -20,4 +21,4 @@ const func = async function (hre) {
 };
 module.exports = func;
 func.tags = ["ExpiringMultiPartyCreator", "emp"];
-func.dependencies = ["Finder", "TokenFactory"];
+func.dependencies = ["Finder", "TokenFactory", "Timer"];
```

Comment:
does this also not have a dependency on `dvm`? I guess as long as there is a `Finder` it will deploy fine but a `Finder` with no `dvm` is not very useful. This is likely what you were referring to yesterday though where you can deploy a contract ecosystem without configuring the finder correctly and that this is a reasonable assumption to let us simplify these scripts.

Reply:
I think our philosophy (or at least the one I'm going with :)) is only specifying dependencies that are _explicitly_ required to be passed in in the constructor. For instance, you may not need the entire dvm to be deployed, you may actually deploy the mock oracle in its place.

This is a weakly held opinion, but in my mind, this is part of keeping the process as light as possible.

Thoughts?

Reply:
Going to go ahead and merge, but yeah, this is somewhat opinionated around keeping things minimal as we discussed yesterday. Let me know if you'd like anything to change in a follow-up!